third_party/move/move-compiler-v2/src/pipeline/ability_checker.rs

```diff
@@ -6,7 +6,8 @@
 //! - liveness analysis and lifetime analysis have been performed
```

Comment:
I believe these are out of date, please update.

Reply:
done